components/PlayerIcon.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react'
+import { Image, Text, View } from 'react-native'
+
+import crown from '../assets/crown.png'
+
+interface Props {
+  name?: String;
+  index: any;
+}
+
+export const PLAYER_IMAGES = [
+  { uri: require('../assets/player-1.png'), color: 'bg-[#f19f3f]' },
+  { uri: require('../assets/player-2.png'), color: 'bg-[#eebebb]' },
+  { uri: require('../assets/player-3.png'), color: 'bg-[#fbe9e7]' },
+  { uri: require('../assets/player-4.png'), color: 'bg-[#F7D2A2]' }
```

Comment:
this is _such_ a nitpick, but if we can make the `color` on this line also lowercase to match the others